src/foam/nanos/referral/tests/ReferralCodeTest.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.referral.test',
```

Comment:
The file system path is tests while the package path is test. 
Can you make the file system path 'test'. Thanks.